src/test/java/posters/cucumber/support/RegisterSupport.java

```diff
@@ -27,24 +22,27 @@ public RegisterSupport(GlobalStorage storage)
     @Given("^user setup: \"([^\"]*)\", \"([^\"]*)\", \"([^\"]*)\", \"([^\"]*)\"$")
     public void setUpUser(String firstName, String lastName, String eMail, String password)
     {
-        // set up user for the clean up steps
+        // create a user for later use (e.g. register or delete)
         storage.user = new User(firstName, lastName, eMail, password);
     };
 
     @Given("^login page is opened after registration$")
     public void registerUserSetup()
     {
         // use the user coming from dependency injection
-        registerUser(storage.user);
+        // registerUser(storage.user);
+        var registerPage = OpenPageFlows.registerPage();
+        registerPage.isExpectedPage();
+        var loginPage = registerPage.sendRegisterForm(storage.user);
+        loginPage.isExpectedPage();
```

Comment:
@oomelianchuk the isExpectedPage method should already have been called by the sendRegisterForm method.
If this is the case please remove the second call.

Reply:
The method is not called anywhere, so I removed it completely